migrations/20200831174542_snapshot.sql

```diff
@@ -31,11 +31,64 @@ DROP TRIGGER IF EXISTS snapshot_notify_delete ON snapshot;
 
 CREATE TRIGGER snapshot_notify_update
 AFTER
-UPDATE ON snapshot FOR EACH ROW EXECUTE PROCEDURE table_snapshot_update_notify();
+UPDATE ON snapshot FOR EACH ROW EXECUTE PROCEDURE table_update_notify();
 
 CREATE TRIGGER snapshot_notify_insert
 AFTER
-INSERT ON snapshot FOR EACH ROW EXECUTE PROCEDURE table_snapshot_update_notify();
+INSERT ON snapshot FOR EACH ROW EXECUTE PROCEDURE table_update_notify();
 
 CREATE TRIGGER snapshot_notify_delete
-AFTER DELETE ON snapshot FOR EACH ROW EXECUTE PROCEDURE table_snapshot_update_notify();
\ No newline at end of file
+AFTER DELETE ON snapshot FOR EACH ROW EXECUTE PROCEDURE table_update_notify();
+
+CREATE TABLE IF NOT EXISTS snapshot_interval (
+  id serial PRIMARY KEY,
+  filesystem_name TEXT NOT NULL,
```

Comment:
Without a constraint it will be possible to create multiple intervals. no?

Reply:
Yes, I still need to address this.

Reply:
Fixed.